node/src/Pos/Block/Logic/Internal.hs

```diff
@@ -153,8 +153,11 @@ normalizeMempool = do
 -- Invariant: all blocks have the same epoch.
 applyBlocksUnsafe
     :: forall ssc ctx m . MonadBlockApply ssc ctx m
-    => OldestFirst NE (Blund ssc) -> Maybe PollModifier -> m ()
-applyBlocksUnsafe blunds pModifier = do
+    => ShouldCallBListener
```

Comment:
Strongly suspect is from https://github.com/input-output-hk/cardano-sl/commit/fee01810a33d73ebf349a0995efc0dd74ced6f58#diff-314075405d88168e3c4ea2e011625453